Microsoft.DotNet.Interactive.PowerShell/Host/PSKernelHost.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.DotNet.Interactive.PowerShell.Host
 {
     public class PSKernelHost : PSHost, IHostSupportsInteractiveSession
     {
-        private const string HostName = "PowerShell Jupyter Host";
+        private const string HostName = "Microsoft.dotnet-interactive";
```

Comment:
fair, but we should still change it to not have Jupyter in the name since .NET Interactive supports more than Jupyter.

I've changed it to "PowerShell .NET Interactive Host"

Reply:
Right, how about just `.NET Interactive Host`?